Algorithm/QCAlgorithm.cs

```diff
@@ -314,48 +310,27 @@ public virtual void OnTick(Dictionary<string, List<Tick>> data)
             //throw new NotImplementedException("OnTick has been made obsolete. Please use OnData(Ticks data) instead.");
         }
 
-        // <summary>
-        // Event - v2.0 TRADEBAR EVENT HANDLER: (Pattern) Basic template for user to override when requesting tradebar data.
-        // </summary>
-        // <param name="data"></param>
-        //public void OnData(TradeBars data)
-        //{
-        //
-        //}
-
-        // <summary>
-        // Event - v2.0 TICK EVENT HANDLER: (Pattern) Basic template for user to override when requesting tick data.
```

Comment:
Do we want to remove this? I think the reason it's been left here is to act as a little doc for derived types.
@jaredbroad  any comment here?